src/pages/workspaces/workspace/notebooks/modals/CloudEnvironmentModal.js

```diff
@@ -0,0 +1,401 @@
+    const isDisabled = !doesCloudEnvForToolExist || !cookieReady || !canCompute || busy || isToolBusy
+    const baseProps = {
+      'aria-label': `Launch ${toolLabel}`,
+      disabled: isDisabled,
+      style: {
+        ...toolButtonStyles,
+        color: isDisabled ? colors.dark(0.7) : colors.accent()
+      },
+      hover: { backgroundColor: colors.accent(0.2) },
+      tooltip: Utils.cond(
+        [doesCloudEnvForToolExist && !isDisabled, () => 'Launch'],
+        [doesCloudEnvForToolExist && isDisabled, () => 'Launch disabled'],
+        [Utils.DEFAULT, () => 'No environment found']
+      ),
+      tooltipDelay: 100
+    }
+
+    return Utils.switchCase(toolLabel,
+      [tools.galaxy.label, () => {
+        return {
+          ...baseProps,
+          href: currentApp?.proxyUrls?.galaxy,
+          onClick: () => {
+            onDismiss()
+            Ajax().Metrics.captureEvent(Events.applicationLaunch, { app: 'Galaxy' })
+          },
+          ...Utils.newTabLinkPropsWithReferrer
+        }
+      }], [Utils.DEFAULT, () => {
+        const applicationLaunchLink = Nav.getLink('workspace-application-launch', { namespace, name: workspaceName, application: toolLabel })
+        return {
+          ...baseProps,
+          href: applicationLaunchLink,
+          onClick: () => {
+            (currentRuntime?.status === 'Stopped' ? () => startApp(toolLabel) : () => {})()
+            onDismiss()
+          }
+        }
+      }]
+    )
+  }
+
+
+  const renderToolButtons = toolLabel => {
+    const doesCloudEnvForToolExist = currentRuntimeTool === toolLabel || (currentApp && toolLabel === tools.galaxy.label)
+    return h(Fragment, [
+      div({ style: toolPanelStyles }, [
+        // Label at the top for each tool
+        div({ style: toolLabelStyles }, [
+          img({
+            src: getToolIcon(toolLabel),
+            style: { height: 20 }
+          }),
+          getCostForTool(toolLabel)
+        ]),
+        // Cloud environment button
+        div({ style: toolButtonDivStyles }, [
+          h(Clickable, {
+            'aria-label': `${toolLabel} Env`,
+            style: {
+              ...toolButtonStyles,
+              color: !isCloudEnvModalDisabled(toolLabel) ? colors.accent() : colors.dark(0.7)
+            },
+            hover: { backgroundColor: colors.accent(0.2) },
+            tooltip: Utils.cond([isCloudEnvModalDisabled(toolLabel), () => 'Edit disabled, processing'],
+              [doesCloudEnvForToolExist, () => 'Edit existing environment'],
+              [!doesCloudEnvForToolExist, () => 'Create new environment (may overwrite existing)']),
+            disabled: isCloudEnvModalDisabled(toolLabel),
+            tooltipDelay: 100,
+            onClick: () => setViewMode(toolLabel)
+          }, [
+            icon('cloudBolt', { size: 20 }), //TODO: why doesn't this icon disable properly color-wise?
```

Comment:
This doesn't look good like the rest of the icons when disabled (it doesn't inherit the color of the parent clickable).... I think its a fact that its an `svg` in the icons foldler and not a font-awesome icon. Any ideas on how to fix this? Its available as a font-awesome pro icon but I think that would require some sort of credentials to pull at yarn install time, which seems like a pain to set up...